libs/ui/packages/native/src/components/cta/QuickAction/QuickActionList/index.tsx

```diff
@@ -19,15 +19,11 @@ const QuickActionList = ({
 }: QuickActionListProps): React.ReactElement => {
   const renderItem = useCallback(
     ({ item, index }: { item: QuickActionButtonProps; index: number }) => {
+      const mr = (index + 1) % numColumns > 0 && data && index !== data.length - 1 ? 4 : 0;
```

Comment:
It would be more readable if we split those even more. For example extract the `(index + 1) % numColumns > 0` in its own variable. Then put `(index + 1) % numColumns > 0 && data && index !== data.length - 1` in another one, maybe called `hasButtonToItsRight` or something like that. It will help a ton to clarify what this does and reduce the cognitive load when maintaining it.